.github/workflows/code_checks.yml

```diff
@@ -90,6 +91,7 @@ jobs:
             if git diff origin/master \
                ':!.github/workflows/code_checks.yml' \
                ':!src/stc/scintilla/' \
+               ':!src/stc/lexilla/' \
```

Comment:
BTW this shouldn't be necessary any longer as git-diff doesn't recurse into submodules by default.

Reply:
Ok, I'll remove them and see if CI succeeds..